channeldb/db.go

```diff
@@ -171,7 +172,7 @@ func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
 	chanDB := &DB{
 		DB:     bdb,
 		dbPath: dbPath,
-		now:    time.Now,
+		Now:    time.Now,
```

Comment:
Yes, this is a shortcut to prevent a test-only dependency from becoming explicit everywhere. Maybe we should indeed make it explicit, but speaking purely functional, this works too?

Reply:
An other option to make it explicit for tests only by adding a `TestOpen(..)` function with an appropriate comment. but in general all dependencies should be injected.

Reply:
I checked out the changes required: https://github.com/lightningnetwork/lnd/compare/master...joostjager:explicit-now-dependency-channeldb

It is an okay change I think, but I don't really want to do it for #3770 now that it is close to merge.